### DISCUSSION TITLE
Unable to Bundle Images in Build Files with Nuxt 3 (Working in Nuxt 2)

### DISCUSSION THREAD

**Discussion by madc** [nuxt/image#1611] [Q&A]
I apologize if this question has already been answered elsewhere. I’ve been searching for a solution to this issue ever since Nuxt 3 was released but haven’t been able to find one yet.

We are using an image provider (Directus) to manage our assets. Serving images directly from the CMS works without any issues.

In our Nuxt 2 setup, we bundled all images into the build files during the generation process. However, I haven’t been able to achieve the same functionality with Nuxt 3.

Is there a way to solve this? I’d be grateful for any input or help.

**Comment by ThornWalli** [comment]
I have the same problem. Have two providers Contentful and Eventbrite, perfect in Nuxt 2.

In Nuxt 3, images are no longer stored locally. 

Has the functionality been removed?

This was an important feature for static pages, alternatively all resources must now be pulled externally, which is not good.

**Comment by puzzle-it-nu** [comment]
I'm using `ipx` and have all the images in the public folder and when building all of them are included, If I add images in production to the public folder then those will be picked up eventually by the app (swr caching).
Is a, I think, rather standard nuxt3 app with @nuxt/image and ipx as a provider, deployed to pm2 behind a haproxy server.
